pkg/users/git_test.go

```diff
@@ -53,6 +53,30 @@ func TestFindUserByLabel(t *testing.T) {
 	assert.Equal(t, labeleduserID, user.Spec.Login)
 }
 
+func TestFindUserByUpperCaseGitUserName(t *testing.T) {
+	t.Parallel()
+	resolver, _, err := prepare(t)
+	assert.NoError(t, err)
+	assert.NoError(t, err)
+
+	gitUser := gits.GitUser{
+		Login: "foo",
+	}
+
+	user, err := resolver.Resolve(&gitUser)
+	assert.NoError(t, err)
+	assert.NotNil(t, user)
```

Comment:
what's about adding some assertions for the actual expected state of the user on top of just checking that it is not _nil_?